run.py

```diff
@@ -83,11 +84,47 @@ def get_single_media(media_name: str):
 
     return render_template('model-movie-tv.html', content=content_data, cast=cast_data, video=video_data, images=image_data, related=related_data['media'][media_name])
 
+@app.route('/about')
+def get_about():
+    return render_template('about.html', commits=get_commits()[0], total_commits=get_commits()[1], issues=get_issues()[0], total_issues=get_issues()[1])
+
+def get_commits():
+    all_commits = 0
+    team = {'stevex196x':0, 'TheSchaft':0, 'melxtru':0, 'aylish19':0, 'connormlewis':0, 'tsukkisuki':0}
+    try:
+        url = 'https://api.github.com/repos/connormlewis/idb/stats/contributors'
+        data = requests.get(url)
+        json_list = data.json()
+        for entry in json_list:
+            total = entry['total']
+            user_name = entry['author']['login']
+            team[user_name] = total
+            all_commits += total
+    except TypeError:
+        return (team, all_commits)
+    return (team, all_commits)
+
+def get_issues():
+    team = {'stevex196x':0, 'TheSchaft':0, 'melxtru':0, 'aylish19':0, 'connormlewis':0, 'tsukkisuki':0}
+    all_issues = 0
+    try:
+        url = 'https://api.github.com/repos/connormlewis/idb/issues?state=all&filter=all'
+        #login_url = 'https://api.github.com/user?access_token=43c3a1848527941be68055b0744de7b0ac2ab2ca'
+        #data = requests.get(login_url)
+        data = requests.get(url)
```

Comment:
in the `requests.get()` call you can do `requests.get(url, headers={'Authorization': 'token ' + os.environ['API_TOKEN']})` to increase the limit to 5000